libs/wingsdk/src/shared/sandbox.ts

```diff
@@ -47,19 +47,29 @@ export class Sandbox {
       );
     }
 
+    let debugShim = "";
+    if (inspectorUrl?.()) {
```

Comment:
Where is this defined? (for my own curiosity)

Reply:
[It comes directly from the inspector builtin module](https://nodejs.org/api/inspector.html#inspectorurl), I mapped it from `url` to be less ambigiguous.